compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -704,10 +768,10 @@ void addMLIRAIRLoweringPasses(OpPassManager &passManager) {
 void addMLIRAIELoweringPasses(OpPassManager &passManager) {
   passManager.addPass(createLowerAffinePass());
   OpPassManager &devicePM = passManager.nest<xilinx::AIE::DeviceOp>();
-  devicePM.addPass(createAMDAIEAssignBufferAddressesBasicPass());
   devicePM.addPass(createAMDAIEAssignLockIDsPass());
-  devicePM.addPass(createAMDAIEAssignBufferDescriptorIDsPass());
   devicePM.addPass(createAMDAIEObjectFifoStatefulTransformPass());
+  devicePM.addPass(createAMDAIEAssignBufferDescriptorIDsPass());
```

Comment:
@makslevental It looks like I need to reorder the passes here to avoid a failure

Reply:
Surprising - not sure how statefultransform collides with bdids but I'm not surprised. Sorry for the hiccup (I reordered these this morning and did think for a second whether it would break but it didn't so 🤷‍♂️).

Reply:
No worries, now we will have an e2e test actually using most of these passes :) so it should bark now when broken